querydsl-core/src/main/java/com/querydsl/core/group/GMultimap.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2015, The Querydsl Team (http://www.querydsl.com/team)
```

Comment:
Is `2015` correct?

Reply:
No, I used GMap as basis for this, and I must have accidentally copied the same license header over.

That does mean though, that there are still several other outdated copyright notices out there 😉